### PR TITLE
ARCHBOM-1213: add CHANGELOG and ADRs

### PR DESCRIPTION
**Pull request by robrap**
NOTE: Includes placeholder ADR 0002 regarding the creation of this repo.

ARCHBOM-1213

### CODE REVIEW

File: docs/decisions/0002-combined-cookiecutter-repo.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
+2. Combined Cookiecutter Repo
+=============================
```
Comment (robrap) [inline docs/decisions/0002-combined-cookiecutter-repo.rst:2]: **Reviewer Note:** This is a placeholder ADR.  Someone can complete it in a separate PR when the docs for this repo are completed.

File: docs/decisions/0002-combined-cookiecutter-repo.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+* cookiecutter-django-ida
+
+Decision
+--------
+
+TODO
+
+Note: The ``cookiecutter-xblock `` will continue to be its own independent cookiecutter repo.
+
+^ Is this true???
```
Comment (jinder1s) [inline docs/decisions/0002-combined-cookiecutter-repo.rst:24]: Yes, xblock was way too different to be included in the layering.

File: docs/decisions/0003-use-changelog.rst [hunk 3]
```diff
@@ -0,0 +1,33 @@
+2. Use CHANGELOG.rst
```
Comment (jinder1s) [inline docs/decisions/0003-use-changelog.rst:1]: This files looks fine, though does the decision to use CHANGELOG need a ADR? Isn't a changelog an edx standard?

**Comment by jinder1s** [comment]
What are your thoughts of having a changelog in each cookiecutter vs having a single changelog for entire repository? I am in favor of one changelog for entire edx-cookiecutters repository.

File: docs/decisions/0001-record-architecture-decisions.rst [hunk 4]
```diff
@@ -0,0 +1,32 @@
+1. Record Architecture Decisions
```
Comment (nedbat) [inline docs/decisions/0001-record-architecture-decisions.rst:1]: Do we need an ADR that says we'll write ADRs?

Comment (robrap) [reply docs/decisions/0001-record-architecture-decisions.rst]: No.  I removed this.

File: docs/decisions/0003-use-changelog.rst (hunk 3 above)
Comment (robrap) [reply docs/decisions/0003-use-changelog.rst]: Not yet.  I would like it to be.  So, we may have a couple of ADRs until we can push forward the Changelog OEP.

File: docs/decisions/0002-combined-cookiecutter-repo.rst (hunk 2 above)
Comment (robrap) [reply docs/decisions/0002-combined-cookiecutter-repo.rst]: 👍

**Comment by robrap** [comment]
@jinder1s: I would say one changelog.  I could imagine different directories for ADRs if we are making template specific decisions, but that is TBD.

**Pull request merged**